packages/react-components/react-charts-preview/stories/LineChart/LineChartDefault.stories.tsx

```diff
@@ -166,51 +173,45 @@ export const LCBasic = (props: ILineChartProps) => {
       />
       <br />
       <Switch
-        label={allowMultipleShapes ? 'Disable multiple shapes for each line' : 'Enable multiple shapes for each line'}
+        label={allowMultipleShapes ? 'Enabled multiple shapes for each line' : 'Disabled multiple shapes for each line'}
         onChange={_onShapeChange}
         checked={allowMultipleShapes}
       />
+      <div style = {{ marginTop: '10px'}}>
       <Switch
-        label={showAxisTitles ? 'Hide axis titles' : 'Show axis titles'}
+        label={showAxisTitles ? 'Show axis titles' : 'Hide axis titles'}
         checked={showAxisTitles}
         onChange={_onToggleAxisTitlesCheckChange}
       />
-      {showAxisTitles && (
-        <div style={rootStyle}>
-          <LineChart
-            culture={window.navigator.language}
-            data={data}
-            legendsOverflowText={'Overflow Items'}
-            yMinValue={200}
-            yMaxValue={301}
-            height={height}
-            width={width}
-            xAxisTickCount={10}
-            allowMultipleShapesForPoints={allowMultipleShapes}
-            enablePerfOptimization={true}
-            enableReflow={true}
-            yAxisTitle={showAxisTitles ? 'Different categories of mail flow' : undefined}
-            xAxisTitle={showAxisTitles ? 'Values of each category' : undefined}
-          />
-        </div>
-      )}
-      {!showAxisTitles && (
-        <div style={rootStyle}>
-          <LineChart
-            culture={window.navigator.language}
-            data={data}
-            legendsOverflowText={'Overflow Items'}
-            yMinValue={200}
-            yMaxValue={301}
-            height={height}
-            width={width}
-            xAxisTickCount={10}
-            enableReflow={true}
-            allowMultipleShapesForPoints={allowMultipleShapes}
-            enablePerfOptimization={true}
-          />
-        </div>
-      )}
+      </div>
+      <div style = {{ marginTop: '10px' }}>
+      <Checkbox
+        label="Use UTC time"
+        checked={useUTC}
+        onChange={_onCheckChange}
+      />
+      </div>
+
+      <div style={rootStyle}>
+        <LineChart
+          // Force rerender when any of the following states change
+          key={`${showAxisTitles}`}
```

Comment:
instead of using this way, are you able to add a hook for axis title

Reply:
Yes, that can be also be done

Reply:
To reduce code length, used this way as done in V8